packages/react/src/progress-indicator/ProgressIndicatorItem.tsx

```diff
@@ -1,144 +1,49 @@
-import { BaseButton } from '../button';
-import { hoverColorMap, ProgressIndicatorBackground } from './utils';
-
-export type ProgressIndicatorItem = (
-	| ProgressIndicatorItemButtonProps
-	| ProgressIndicatorItemLinkProps
-) & {
-	label: string;
-	isActive?: boolean;
-};
-
-export type ProgressIndicatorItemStatus =
-	| 'blocked'
-	| 'doing'
-	| 'done'
-	| 'error'
-	| 'saved'
-	| 'started'
-	| 'todo';
-
-export interface ProgressIndicatorItemLinkProps extends LinkProps {
-	/** Determines the background colour of the progress indicator item. */
-	background?: ProgressIndicatorBackground;
-	/** Determines what style the item displays as.
-	 *
-	 * `blocked` = This step is not yet available for the user to start.
-	 *
-	 * `doing` = **(deprecated)** This step has been started and the user is actively working on this step (Use `started` with `isActive: true` instead).
-	 *
-	 * `done` = This step is complete. The user has submitted valid data and it has been verified (where necessary).
-	 *
-	 * `error` = This step has an error that requires attention.
-	 *
-	 * `saved` = This step has valid saved data that can still be changed.
-	 *
-	 * `started` = This step has been started and still needs to be completed.
-	 *
-	 * `todo` = This step is available for the user to start.
-	 */
-	status: ProgressIndicatorItemStatus;
-	/** Set this item as currently active. Only supporte for the `blocked`, `started`, and `doing` statuses */
-	isActive?: boolean;
-}
-
-export const ProgressIndicatorItemLink = ({
-	children,
-	isActive,
-	status,
-	...props
-}: ProgressIndicatorItemLinkProps) => (
-	<ProgressIndicatorItem
-		as={TextLink}
-		isActive={Boolean(isActive)}
-		status={status}
-		{...props}
-	>
-		{children}
-	</ProgressIndicatorItem>
-);
-
-export type ProgressIndicatorItemButtonProps =
-	ButtonHTMLAttributes<HTMLButtonElement> & {
-		background?: ProgressIndicatorBackground;
-		status: ProgressIndicatorItemStatus;
-		isActive: boolean;
-	};
-
-export const ProgressIndicatorItemButton = ({
-	children,
-	...props
-}: ProgressIndicatorItemButtonProps) => (
-	<ProgressIndicatorItem as={BaseButton} {...props}>
-		{children}
-	</ProgressIndicatorItem>
-);
-
-export const activeProgressStatusAllowedList: Array<ProgressIndicatorItemStatus> =
-	['blocked', 'doing', 'started'];
-
-export const activeProgressStatusBlockedList: Array<ProgressIndicatorItemStatus> =
-	['done', 'error', 'saved', 'todo'];
-export const allProgressStatuses: Array<ProgressIndicatorItemStatus> = [
-	...activeProgressStatusAllowedList,
-	...activeProgressStatusBlockedList,
-];
+	hoverColorMap,
+	ProgressIndicatorBackground,
+	progressIndicatorItemRingDataAttr,
+	ProgressIndicatorItemStatus,
+	progressIndicatorItemTextContainerDataAttr,
+	progressIndicatorItemTextDataAttr,
+	progressIndicatorItemTimelineDataAttr,
+	ProgressIndicatorItemWithLevelTwoItems,
+	statusMap,
+} from './utils';
 
 type ProgressIndicatorItemProps = PropsWithChildren<{
+	status: ProgressIndicatorItemStatus;
```

Comment:
Alphabetise, as you seem to be doing that

Reply:
- [x] alphabetise